crd/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+use kube_derive::CustomResource;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use stackable_operator::Crd;
+
+#[derive(
+    Clone, CustomResource, Debug, Default, Deserialize, Eq, Hash, JsonSchema, PartialEq, Serialize,
+)]
+#[kube(
+    group = "opa.stackable.tech",
+    version = "v1",
+    kind = "OpaServer",
+    shortname = "os",
```

Comment:
nit: wouldn't "opa" be the obvious choice here?

Reply:
Yeah, any changes in the group name then? 
"opa.opa.stackable.tech" is not my favorite :)